control-plane-operator/controllers/hostedcontrolplane/etcd/params.go

```diff
@@ -13,6 +13,10 @@ import (
 
 const (
 	EtcdSTSQuotaBackendSize = 8 * 1024 * 1024 * 1024
+	// PodSafeToEvictKey is an annotation used by the CA operator which makes sure
+	// that all the pods annotated with it, could be drained properly.
+	// source https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node
+	PodSafeToEvictKey = "cluster-autoscaler.kubernetes.io/safe-to-evict"
```

Comment:
does this annotation somehow skip PDBs? If so that's highly undesired and will need to use safe-to-evict-local-volumes https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node

How do those annotations differ?

Reply:
to clarify, it's not only about being specific on volumes. It's that we don't want this behaviour https://github.com/kubernetes/autoscaler/blob/d1740df93b76d99bf9302ad0c62978deb0ec1d5b/cluster-autoscaler/utils/drain/drain.go#L39-L40